### PR TITLE
fix: card assignee not updated after quick assign

### PR DESCRIPTION
**Pull request by juniusfree**
## :pushpin: References

- **Wonder issue:** https://app.wonderverse.xyz/organization/wonderverse/boards?task=75498103216538021&entity=task
- **Related pull-requests:** n/a

## :blue_book: Description

Card assignee not updated after quick assign

## :movie_camera: Screenshot or Video

https://user-images.githubusercontent.com/8164667/207796637-d2ba98c6-e4b0-4ca3-806d-c1b65383404c.mp4

## :cake: Checklist:

- [x] I self-reviewed my changes
- [x] My changes follow the style guide: https://creative-earth-33e.notion.site/TT-Wonder-Style-guide-4702a45133374148953bfcaf584120b7
- [x] I tested my changes in Chrome

### CODE REVIEW

File: wondrous-app/utils/helpers.ts [hunk 1]
```diff
@@ -92,17 +92,18 @@ export const hasCreateTaskPermission = ({ userPermissionsContext, orgId = undefi
   return permissions.some((i) => [PERMISSIONS.FULL_ACCESS, PERMISSIONS.CREATE_TASK].includes(i));
 };
 
-export const transformTaskToTaskCard = (task, extraData) => ({
+export const transformTaskToTaskCard = (task, extraData = null) => ({
   ...task,
-  assigneeUsername: task?.assignee?.username,
-  assigneeProfilePicture: task?.assignee?.profilePicture,
+  assigneeUsername: task?.assignee?.username || null,
```
Comment (andywong418) [inline wondrous-app/utils/helpers.ts:97]: Why do we need to set this to null?

File: wondrous-app/components/Common/TaskViewModal/taskViewModalFields.tsx [hunk 2]
```diff
@@ -299,10 +279,7 @@ export function AssigneeField({
                 variables: {
                   taskId: fetchedTask?.id,
                 },
-                onCompleted: (data) => {
-                  const task = data?.removeTaskAssignee;
-                  updateBoard({ task, setFetchedTask, boardColumns });
```
Comment (andywong418) [inline wondrous-app/components/Common/TaskViewModal/taskViewModalFields.tsx:304]: Will this update the board after updating in the card tho?

Comment (andywong418) [reply wondrous-app/components/Common/TaskViewModal/taskViewModalFields.tsx:304]: Actually seems like it does!

File: wondrous-app/utils/helpers.ts (hunk 1 above)
Comment (juniusfree) [reply wondrous-app/utils/helpers.ts:97]: We need to set `null` to override the value of `assigneeUsername` and `assigneeProfilePicture` when the `assignee` is removed.

File: wondrous-app/components/Common/TaskViewModal/taskViewModalFields.tsx (hunk 2 above)
Comment (juniusfree) [reply wondrous-app/components/Common/TaskViewModal/taskViewModalFields.tsx:304]: Yes, according to the docs

> All subscribers to the Apollo Client cache (including all active queries) see this change and update your application's UI accordingly.
https://www.apollographql.com/docs/react/caching/cache-interaction/#writefragment

**Review by andywong418 — APPROVED** [review]

**Pull request merged**